lib/cluster-providers/mng-cluster-provider.ts

```diff
@@ -1,7 +1,7 @@
 import { Construct } from "@aws-cdk/core";
 import * as eks from "@aws-cdk/aws-eks";
 import { AutoScalingGroup } from "@aws-cdk/aws-autoscaling";
-import { InstanceType, IVpc, SubnetSelection, SubnetType } from "@aws-cdk/aws-ec2";
+import { IMachineImage, InstanceType, IVpc, LaunchTemplate, SubnetSelection, SubnetType, UserData } from "@aws-cdk/aws-ec2";
```

Comment:
Minor: please use wildcard import if the number of imports exceeds 3 (it is an internal convention atm, we will add it to lint): ` import * as ec2 from ...`

Reply:
Fixed. Thanks.